internal/daemonConfig/config.go

```diff
@@ -0,0 +1,77 @@
+package daemonConfig
+
+import (
+	"errors"
+	"github.com/creasty/defaults"
+	"github.com/goccy/go-yaml"
+	icingadbConfig "github.com/icinga/icingadb/pkg/config"
+	"os"
+)
+
+type ConfigFile struct {
+	Listen           string                  `yaml:"listen" default:"localhost:5680"`
+	DebugPassword    string                  `yaml:"debug-password"`
+	ChannelPluginDir string                  `yaml:"channel-plugin-dir" default:"/usr/libexec/icinga-notifications/channel"`
+	Icingaweb2URL    string                  `yaml:"icingaweb2-url"`
+	Database         icingadbConfig.Database `yaml:"database"`
+	Logging          icingadbConfig.Logging  `yaml:"logging"`
+}
+
+var config *ConfigFile
```

Comment:
If this variable is used like a singleton, maybe add a short comment saying so. Currently I'm seeing access from both `Load` and `Config`.